nilearn/glm/tests/test_dmtx.py

```diff
@@ -12,15 +12,11 @@
 
 from nibabel.tmpdirs import InTemporaryDirectory
 from numpy.testing import (
-    assert_almost_equal, assert_array_equal, assert_array_almost_equal,
-    assert_equal)
+    assert_almost_equal, assert_array_equal, assert_array_almost_equal)
 
-from nilearn.glm.first_level.design_matrix import (_convolve_regressors,
-                                                   _cosine_drift,
-                                                   check_design_matrix,
-                                                   make_first_level_design_matrix,
-                                                   make_second_level_design_matrix,
-                                                   )
+from nilearn.glm.first_level.design_matrix import \
```

Comment:
I changed the style here. Would you prefer one import per line?

Reply:
I don't think this is covered in the style guide, but I personally prefer `black`'s style. 

So it would be more like

```python
from X import (
    x,
    y,
    z,
)
```

Reply:
Done, thanks